modules/mono/glue/GodotSharp/GodotSharp/Core/Bridge/ScriptManagerBridge.types.cs

```diff
@@ -13,7 +14,7 @@ public static partial class ScriptManagerBridge
 {
     private class ScriptTypeBiMap
     {
-        public readonly object ReadWriteLock = new();
+        public ReaderWriterLockSlim ReadWriteLock = new(LockRecursionPolicy.SupportsRecursion);
```

Comment:
Shouldn't need to remove the `readonly` modifier, since this field is never re-assigned.

```suggestion
        public readonly ReaderWriterLockSlim ReadWriteLock = new(LockRecursionPolicy.SupportsRecursion);
```

Reply:
Done.